docs/serving/spec/knative-api-specification-1.0.md

```diff
@@ -1,3 +0,0 @@
-# Knative Serving API Specification
-
-This file has been moved to the [Knative Specs Repository](https://github.com/knative/specs/blob/main/specs/serving/knative-api-specification-1.0.md)
```

Comment:
I think we left this file as a pointer for external links. I'm not seeing the redirect right now, so we might want to leave this in place?

https://github.com/knative/website/blob/main/static/_redirects#L3